config/checker/java.util.astub

```diff
@@ -27,6 +27,10 @@ class AtomicReference<V> {
 
 package java.util.stream;
 
+interface Stream<T> {
+  <@Nullable R> Stream<@Nullable R> map(Function<? super T, ? extends @Nullable R> var1);
```

Comment:
Maybe:

```java
<@PolyNull R> Stream<@PolyNull R> map(Function<? super T, ? extends @PolyNull R> var1);
```

Reply:
Then it should behave as if there were two overloaded aspects of `map`:

`Maybe:

```java
<@Nullable R> Stream<@Nullable R> map(Function<? super T, ? extends @Nullable R> var1);
<@NonNull R> Stream<@NonNull R> map(Function<? super T, ? extends @NonNull R> var1);
```

Reply:
None of them works :c resolved it for now by returning `""` from `getPrunedRef` instead of `null` :trollface:

Reply:
:frowning_face: